### PR TITLE
Support new AR 5 + Dirty methods

### PR DESCRIPTION
**Pull request by drobny**
Fixes https://github.com/lwe/simple_enum/issues/146

### CODE REVIEW

File: lib/simple_enum/attribute.rb [hunk 1]
```diff
@@ -72,6 +72,9 @@ def generate_enum_dirty_methods_for(enum, accessor)
       simple_enum_module.module_eval do
         define_method("#{accessor}_changed?") { accessor.changed?(self) }
```
Comment (lwe) [inline lib/simple_enum/attribute.rb]: should we also mark these as deprecated?

**Comment by ykamez** [comment]
Gemfile might be causing error in Ci.

```
  18) SimpleEnum::Attribute generate_enum_scope_methods_for without scope method does not add .males nor .females
     Failure/Error: Unable to find matching line from backtrace
     Gem::LoadError:
       Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
     # ./spec/support/active_record_support.rb:3:in `connection'
     # ./spec/support/active_record_support.rb:12:in `reset_active_record'
     # ./spec/support/active_record_support.rb:8:in `block in included'
```

ref: https://travis-ci.org/lwe/simple_enum/jobs/536659811

**Comment by ykamez** [comment]
Also I've updated README. https://github.com/drobny/simple_enum/pull/1

**Comment by ykamez** [comment]
@lwe Could you release this pr?

**Comment by lwe** [comment]
Hey @ykamez, sorry for the delay, just merged it!

**Comment by ykamez** [comment]
No problem. Thanks 😄

**Pull request merged**